update.php

```diff
@@ -45,6 +45,7 @@
 use Friendica\Core\Update;
 use Friendica\Core\Worker;
 use Friendica\Database\Database;
+use Friendica\Database\DBStructure;
```

Comment:
This exact `use` statement is present two lines below so I don't think this is the reason why you had a "Class not found" error.

Reply:
The tests are failing because of the duplicate `use` statements.